tjdashboard/code/site/views/widgets/tmpl/default.xml

```diff
@@ -11,7 +11,7 @@
             label="COM_TJDASHBOARD_FIELDSET_ITEM_ID_SELECT_LABEL">
             <field
                 name="dashboard_id"
-                query="SELECT `dashboard_id`, `title` FROM #__tj_dashboards ORDER BY `dashboard_id`"
+                query="SELECT `dashboard_id`, `title` FROM #__tj_dashboards ORDER BY `ordering` where `state`=1"
```

Comment:
Over here, I think ordering by `title asc` or `created date desc` might be better. What do you think ?

Reply:
Here I see , We are not even showing title of dashboard as the query selecting only dashboard id If we are going to show dashboards title there we should add title in select. 

In my opinion ordering column make sense, If you check with categories and article case they shows categories in article creation and it shows according to ordering column If you change order of categories it reflect in article form as well.